### DISCUSSION TITLE
How to use different batch size for training and validation

### DISCUSSION THREAD

**Discussion by wzr0108** [open-mmlab/mmclassification#1662] [Q&A] [answered]
How to use different batch size for training and validation

**Comment by Ezra-Yu** [comment]
just modify the `train_dataloader.batch_size` or `test_dataloader.batch_size` 
eg. https://github.com/open-mmlab/mmpretrain/blob/4dd8a861456a88966f1672060c1f7b24f05ec363/configs/_base_/datasets/coco_caption.py#L37

**Reply by wzr0108** [reply]
i use mmcls0.x, there is no train_dataloader in the config

**Reply by Ezra-Yu** [reply]
https://github.com/open-mmlab/mmpretrain/blob/748ab7aa7d76dc35939e09a59632a8c3f073c8bc/configs/_base_/datasets/imagenet_bs64.py#L23

**Reply by wzr0108** [reply]
when i set samples_per_gpu=64, the train bs and val bs are both 64 on single gpu. How to use different batch size for training and validation

**Reply by Ezra-Yu** [reply, marked as answer]
This was a flaw in the previous design. In the new version, we can now set different batch sizes during the train/test stages.
